### PR TITLE
I/O task

### PR DESCRIPTION
**Pull request by Tiger656**

### CODE REVIEW

File: src/test/java/com/epam/mjc/io/FileReaderTest.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    private final static Path READER_CLASS = Path.of("src/main/java/com/epam/mjc/io/FileReader.java");
+    private static final String PROFILE_PARAM_VALUE_DELIMETER = ": ";
+    private static final String SOURCE_CODE_LOCATION = "src/main/java";
+
+
+    @Test
+    public void testProgramReadsDataCorrectly () {
+        FileReader fileReader = new FileReader();
+        Profile actual = fileReader.getDataFromFile(new File(TEST_FILE.toUri()));
+        Profile expected = readFile(TEST_FILE);
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/io/FileReaderTest.java]: Sorry, but student can look into this file and copy logic from here.

File: src/test/java/com/epam/mjc/io/FileReaderTest.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+    private void assertSourceHasClosing(Path path) {
+        String sourceCode = readFileIntoString(path);
+        assertTrue(sourceCode.contains("try (") || sourceCode.contains("try(") || sourceCode.contains(".close()"));
+    }
+
+    private void assertSourceWithoutProhibitedLibraries(Path path) {
+        String sourceCode = readFileIntoString(path);
+        assertFalse(sourceCode.contains(".nio"));
+        assertFalse(sourceCode.contains("Files"));
```
Comment (egoist301) [inline src/test/java/com/epam/mjc/io/FileReaderTest.java]: Do you really need check `Files`? As I can see, `Files` is class from `nio` package.

Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: I agree

File: src/test/java/com/epam/mjc/io/FileReaderTest.java (hunk 1 above)
Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: Fixed

File: src/test/java/com/epam/mjc/io/FileReaderTest.java (hunk 2 above)
Comment (Tiger656) [reply src/test/java/com/epam/mjc/io/FileReaderTest.java]: Fixed

**Review by egoist301 — APPROVED** [review]

**Pull request merged**